tests/scripts/all.sh

```diff
@@ -873,7 +873,12 @@ component_check_doxygen_warnings () {
     tests/scripts/doxygen.sh
 }
 
+component_check_test_requires_psa_disabled () {
+  msg "Check: tests requiring PSA to be disabled"
 
+  not grep -n 'depends.*!MBEDTLS_USE_PSA_CRYPTO' -R tests/suites/
```

Comment:
Minor: `grep -n -R ...` (options before arguments for better compatibility with non-GNU versions of grep.